.github/workflows/rust.yml

```diff
@@ -25,27 +25,30 @@ jobs:
 
     - name: fmt
       run: |
+        cargo generate-lockfile --verbose && cargo update -p funty --precise "1.1.0" --verbose
```

Comment:
@ascjones this is a bit terrible – do you know of a better way? (Context: https://github.com/myrrlyn/funty/issues/3)

Reply:
I'd be okay with adding an explicit pin to the `Cargo.toml`, I've seen other big projects e.g. https://github.com/Geal/nom/commit/dba1e0a9937067201d7e706f7a6043834d2acfc7 just do that.

Reply:
I did that [here](https://github.com/paritytech/parity-scale-codec/pull/250) but there is not consensus for that approach.

Reply:
> an explicit pin to the `Cargo.toml`,

Also, there'd be three different `Cargo.toml`s to fix, so there's an argument for keeping all the ugliness here in one place.

Reply:
I'm not sure what a nice solution to this is, off the top of my head.

Reply:
Me neither.